go/ct/rlz/conditions_test.go

```diff
@@ -463,3 +463,26 @@ func TestCondition_RestrictTransientStorageAndCheck(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkCondition_IsAddressWarmCheckWarm(b *testing.B) {
+	state := st.NewState(st.NewCode([]byte{}))
+	state.Accounts.MarkWarm(NewAddress(NewU256(42)))
+	state.Stack.Push(NewU256(42))
+	condition := IsAddressWarm(Param(0))
+
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		condition.Check(state)
+	}
+}
+
+func BenchmarkCondition_IsAddressWarmCheckCold(b *testing.B) {
+	state := st.NewState(st.NewCode([]byte{}))
+	state.Accounts.MarkWarm(NewAddress(NewU256(42)))
+	condition := IsAddressWarm(Param(0))
```

Comment:
This test case does not set up `Param(0)` on the stack, so what it actually tests is how fast it can check that the stack is empty.